PWGCF/FemtoUniverse/Core/FemtoUniverseParticleHisto.h

```diff
@@ -339,18 +341,25 @@ class FemtoUniverseParticleHisto
   template <bool isMC, bool isDebug, typename T>
   void fillQA(T const& part)
   {
+    fillQA2<isMC, isDebug, T>(part, HIST(o2::aod::femtouniverseparticle::ParticleTypeName[mParticleType]) + HIST(mFolderSuffix[mFolderSuffixType]));
```

Comment:
Do you need `fillQA2` if `fillQA` does nothing apart from calling `fillQA2`?
If yes, could you give more descriptive names? E.g., `fillQA` for the outer function and `doFillQA` for the inner? This at least suggests which function wraps which one.

Reply:
So the idea is to be able to plot QA plots for each V0 particle for V0-V0 combinations without affecting the default settings. fillQA2 was added to accept a general histogram path. The original fillQA is retained because other analysis still use it. fillQA uses the fillQA2 with the template h set to the original default behavior.